configs/apps/ufs-weather-model/spack.yaml

```diff
@@ -1,9 +1,8 @@
-# Template spack.yaml for apps
 spack:
   concretization: separately
   view: false
 
-  @CONFIG_INCLUDES@
+  include: []
```

Comment:
Why does only the ufs-weather-model have `include: []`, but none of the other apps?

Reply:
No good reason. Something I was playing with how the yaml keys were being inserted.

Reply:
I added the include everywhere just so the includes placed near the top of the file.